### ISSUE TITLE
Your basic example how to use freezed does not work.

### ISSUE THREAD

**Issue by lukaszciastko** [rrousselGit/freezed#694]
I wanted to give freezed a go and copied the example code for a Person model from your documentation.
I placed the code in model.dart file and modified it accordingly.

```
import 'package:freezed_annotation/freezed_annotation.dart';
import 'package:flutter/foundation.dart';

part 'models.freezed.dart';
// optional: Since our Person class is serializable, we must add this line.
// But if Person was not serializable, we could skip it.
part 'models.g.dart';

@freezed
class Person with _$Person {
  const factory Person({
    required String firstName,
    required String lastName,
    required int age,
  }) = _Person;

  factory Person.fromJson(Map<String, Object?> json)
  => _$PersonFromJson(json);
}

```

 After generating code, I'm getting two errors:

_Target of URI hasn't been generated: 'models.g.dar._

And, inside models.freezed.dart:

_The method '_$$_PersonToJson' isn't defined for the type '_$_Person'._

I would expect the example code to work.

**Comment by rrousselGit** [comment]
Sounds like you didn't install json_serializable

**Comment by lukaszciastko** [comment]
> Sounds like you didn't install json_serializable

You are right, but if this is required, then it should be mentioned at the beginning of the documentation - not at the very bottom. People who are trying to start with freezed would normally read the documentation from the top and try the example first. And they would fail immediately. Not everyone would read to the end if what they tried first didn't work.
